.eslintrc.js

```diff
@@ -83,6 +84,7 @@ module.exports = {
             files: ['**/expressions/expressions-destructuringassignment.js'],
             rules: {
                 'one-var-declaration-per-line': 'off',
+                'prefer-const': 'off',
```

Comment:
I think the `destructuring: all` [option](https://eslint.org/docs/rules/prefer-const#destructuring) would solve the issue but simply turning off the rule is fine as well in my opinion.